### ISSUE TITLE
Question about self-supervised models

### ISSUE THREAD

**Issue by pengzhiliang** [bethgelab/model-vs-human#18]
Hello,  thanks for  the great toolbox~ 

I'm little confusion about the results on self-supervised models, like SimCLR. It **doesn't** have the class-specifical classifier on ImageNet-1k. 
So how do you load the classifier weight? load an extra-trained classifier (like linear-probing protocol) or a fully-trained network (like end-to-end fine-tuning protocol)?

And another question:
If I just want to test the shape & texture accuracy,  mentioned in [Intriguing Properties of Vision Transformers](https://github.com/Muzammal-Naseer/Intriguing-Properties-of-Vision-Transformers), which dataset type should I choose? 

Thank you very much~

**Comment by rgeirhos** [comment]
Hi @pengzhiliang thanks for your interest!

Regarding texture-shape bias, this is the dataset name: `cue-conflict`.

Regarding SimCLR, the model is loaded here: https://github.com/bethgelab/model-vs-human/blob/master/modelvshuman/models/pytorch/simclr/zoo/simclr.py which includes an extra linear classifier trained by the authors. The SimCLR model is not fine-tuned on ImageNet, it is purely trained in a SSL fashion except for the linear classifier.

Let me know if you have any further questions!

**Comment by pengzhiliang** [comment]
I see.
Thank u~

**Comment by pengzhiliang** [comment]
I'm sorry that I am back again.

I'd like to know whether I need to prepare all the OOD datasets, because I didn't find the relevant transform code under the datasets directory.
So I'm not sure if this repo will generate different types of datasets in real time.

I'm sorry that not very familiar with this repo at the moment, and I don't have a machine to run it for now.

Thank u~

**Comment by rgeirhos** [comment]
No worries, I'm here to help! The best start would probably be to execute the example code here: https://github.com/bethgelab/model-vs-human/blob/master/examples/evaluate.py (you can modify line 8 to simply `datasets = ["cue-conflict"]`. This will automatically download the dataset and the dataset will be preprocessed accordingly using the transforms specified here: https://github.com/bethgelab/model-vs-human/blob/master/modelvshuman/datasets/dataloaders.py#L41

Does this answer your question?

**Comment by pengzhiliang** [comment]
Yes.

It really helps a lot.
And you are so kind~
Thank u~
